open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/ffdc/SearchIntegratorAuditCode.java

```diff
@@ -40,6 +40,13 @@ public enum SearchIntegratorAuditCode implements AuditLogMessageSet {
             "The integration connector is running but does not have a context.  This is a timing issue in the integration daemon.",
             "Gather information about the connector's configuration, the types of metadata it was integrating, the audit log messages " +
                     "from the integration daemon and its partner metadata server.  Then contact the Egeria community to get help."),
+
+    REGISTER_CATALOG_ISTENER("OMIS-SEARCH-INTEGRATOR-0004",
```

Comment:
fixed thank you

Reply:
I think it's a good idea to check if there's recent activity in the module, and if so consider if it might have an impact . Less so in my view for small, isolated change. New events etc - more so as that affects behaviour. I think this helps build community and engagement.

regardless, Github should automatically assign the code owner a review. That should be the formal way of ensuring they are notified and have the opportunity to review. Since I don't see any reviews assigned I suspect the CODEOWNER is not set (cc: @bogdan-sava ) so I would consider updating /CODEOWNERS accordingly